CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [4.0.0-beta]
+- add sentry
```

Comment:
we have list of supported features in readme file (`Supports` section)
please add sentry there. 
we don't need separate changelog file. the changelog will be based on commits and will be attached for each release  (see https://github.com/django-stars/frontend-skeleton/releases)